files/fr/web/api/window/getselection/index.md

```diff
@@ -39,8 +39,8 @@ DOM Level 0. Ne fait partie d'aucune spécification.
 
 ## Voir aussi
 
-- [Selection API](../Selection_API)
-- [`Selection`](../Selection)
-- [`Range`](../Range)
-- [`Document.getSelection()`](../Document/getSelection)
-- [`HTMLInputElement.setSelectionRange()`](../HTMLInputElement/setSelectionRange)
+- [Selection API](/fr/docs/Web/API/Selection)
+- {{domxref("Selection")}}
+- {{domxref("Range")}}
+- {{domxref("Document.getSelection()")}}
+- {{domxref("HTMLInputElement.setSelectionRange()")}}
```

Comment:
```suggestion
- [`Selection`](/fr/docs/Web/API/Selection)
- [`Range`](/fr/docs/Web/API/Range)
- [`Document.getSelection()`](/fr/docs/Web/API/Document/getSelection)
- [`HTMLInputElement.setSelectionRange()`](/fr/docs/Web/API/HTMLInputElement/setSelectionRange)
```

Reply:
we try to remove xref macros as we go in `fr`

Reply:
No problem, but I think I copied this from upstream that is still using macros on this  page